specification/devcenter/DevCenter/python-client.tsp

```diff
@@ -0,0 +1,56 @@
+import "./environments/routes.tsp";
+import "./devcenter/routes.tsp";
+import "./devbox/routes.tsp";
```

Comment:
Same here this should be just main.tsp

Reply:
Also, I noticed that there were some python renames in the other client.tsp, if this file is the one we're using for python customizations then those renames should be moved here.